pkgs/os-specific/darwin/utm/default.nix

```diff
@@ -33,6 +34,11 @@ stdenvNoCC.mkDerivation rec {
     runHook postInstall
   '';
 
+  passthru.updateScript = gitUpdater {
+    url = "https://github.com/utmapp/UTM.git";
+    rev-prefix = "v";
+  };
+
```

Comment:
My concern with using the `gitUpdater` directly is that UTM tends to push tags for minor releases that are then marked as "pre-release"; i.e. `4.2.4` was the first stable `4.2.x` release: https://github.com/utmapp/UTM/releases

I'm not familiar with how the updater script is actually triggered by r-ryantm; repology distinguishes between stable/pre-release tags already so maybe this isn't an issue?

Reply:
I don't think `list-git-tags` distinguishes between `4.2.3` and `4.2.4`, so that will be problematic.

Reply:
Using `nix-update-script` instead.

Reply:
Does `nix-update-script` distinguish pre-releases?

Maybe I'm missing something but it [doesn't look like it does](https://github.com/Mic92/nix-update/blob/6639ad481c667be11b50fd482b30b4702922df63/nix_update/version/github.py#L13-L36); the Atom feeds it parses ([UTM's as an example](https://github.com/utmapp/UTM/releases.atom)) don't seem to encode this information.

Reply:
Probably worth promoting the expression to an general updater.